contracts/eth/nearbridge/contracts/NearBridge.sol

```diff
@@ -79,27 +80,36 @@ contract NearBridge is INearBridge {
         Ed25519 ed,
         uint256 lockEthAmount_,
         uint256 lockDuration_,
-        uint256 replaceDuration_
+        uint256 replaceDuration_,
+        address admin_
     ) public {
+        require(replaceDuration_ > lockDuration_.mul(1000000000));
```

Comment:
Currently, because of this line, the unit tests fail with the following error:

```
Error: while migrating NearBridge: The contract code couldn't be stored, please check your gas limit.
```

(despite what the message says, it is revert, not out of gas). Please fix this.

Reply:
Fixed